### DISCUSSION TITLE
Wazuh vulnerability scans

### DISCUSSION THREAD

**Discussion by Stephen-Ridgway** [Security-Onion-Solutions/securityonion#4875] [General] [answered]
Hi!

Has anyone got Wazuh vulnerability scans working? 

SR

**Comment by ThreeEyedRaven-99** [comment, marked as answer]
Hi SR!

For Wazuh Vulnerability Scan you can follow these steps:

1. Download NVD Database in SO and extract the NVD database in a directory. 
2. Bind the NVD directory in /opt/so/saltstack/default/salt/wazuh/init.sls (for example: if your NVD Database directory is in /root/Downloads/NVD/feeds, you can do like this in init.sls file:  - /root/Downloads/NVD/feeds:/mnt/feeds
3. In wazuh config file add vulnerability-detector section and then you can mention the path of the NVD feeds:
<provider name="nvd">
    <enabled>yes</enabled>
    <path>/mnt/feeds/nvd-feed.*json.gz$</path>
    <update_interval>1h</update_interval>
</provider>

4. restart wazuh: so-wazuh-restart

5. Once vulnerability scan finishes you will see the data in alert section in the Kibana - or you can search for data.vulnerability in discover  section

Please excuse my poor English :)
